src/app/header/components/TermButtons.tsx

```diff
@@ -0,0 +1,42 @@
+import { Button, ButtonGroup } from '@chakra-ui/react';
+import React, { useCallback, useState } from 'react';
+
+import { useTerm } from '../../shared/hooks/useTerm';
+import { getCurrentTerms, getReadableTerm } from '../../shared/utils/terms';
+
+export function TermButtons(): JSX.Element {
+  const { setTerm } = useTerm();
+  const [status, setStatus] = useState([true, false, false]);
+
+  const terms = getCurrentTerms();
+
+  const onClick = useCallback(
+    (event: React.MouseEvent<HTMLElement, MouseEvent>, name: string, i: number) => {
+      event.preventDefault();
+      setTerm(name);
+      const status = [false, false, false];
+      status[i] = true;
+      setStatus(status);
+    },
+    [setTerm]
+  );
+
+  return (
+    <ButtonGroup spacing="0" isAttached>
+      {terms.map((term, i) => {
+        return (
+          <Button
+            key={i}
+            name={term}
+            isActive={status[i]}
+            onClick={(e) => onClick(e, term, i)}
```

Comment:
It seems like a bad idea to use `useCallback` but also have it be making an anonymous function each time. It would be nice to get rid of either passing in `term` or `i`. Using `date-*` atrributes and accessing them through `e.currentTarget.getAttribute('date-*'')` might be a way around that. then you can just pass `onClick` directly (and avoid making a new function each time (and get max value for that `useCallback` imo)